src/connectedk8s/HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+1.8.0
+++++++
+* Add support for api version 07-01-2024
```

Comment:
Unless there was a real requirement to keep these as a single change, it would have been nice to see these are two PRs so that the specific changes for the two were clearly communicated.